fwmt-job-firehose/rm_builder.py

```diff
@@ -76,13 +93,28 @@ def construct_RM_action_request(id, address_data, contact_data):
     events  = ET.SubElement(req, 'events') # mandatory
     # event_1 = ET.SubElement(events, 'event', text = 'foo')
 
-    exerciseRef       = ET.SubElement(req, 'exerciseRef')
-    userDescription   = ET.SubElement(req, 'userDescription')
-    surveyName        = ET.SubElement(req, 'surveyName')
-    surveyRef         = ET.SubElement(req, 'surveyRef') # 'HH', 'CCS' or 'CE' to match a job service component
-    surveyRef.text    = 'HH'
-    returnByDate      = ET.SubElement(req, 'returnByDate')
-    returnByDate.text = '01/01/1999'
+    exerciseRef             = ET.SubElement(req, 'exerciseRef')
+    userDescription         = ET.SubElement(req, 'userDescription')
+    surveyName              = ET.SubElement(req, 'surveyName')
+    surveyRef               = ET.SubElement(req, 'surveyRef') # 'HH', 'CCS' or 'CE' to match a job service component
+    surveyRef.text          = 'HH'
+    returnByDate            = ET.SubElement(req, 'returnByDate')
+    returnByDate.text       = '01/01/1999'
+    addressType             = ET.SubElement(req, 'addressType')
+    addressType.text       = 'HH'
```

Comment:
The alignment has been altered somewhat, but only bother fixing it if you can have your editor do it for you.
If it's left as-is, I'll sort it next time I'm in this file.